src/structures/AnonymousGuild.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const BaseGuild = require('./BaseGuild');
+const { VerificationLevels, NSFWLevels } = require('../util/Constants');
+
+/**
+ * Bundles common attributes between guild classes
```

Comment:
You could add a mention to the two deriving classes to make it easier to navigate the docs (see `BaseGuild`)